### ISSUE TITLE
Mass accuracy error and self.version adjustment 

### ISSUE THREAD

**Issue by ahi-ed** [Tarskin/LaCyTools#45]
**Describe the bug**
If program runs through Python 3 (product version 3.8.2) the output values are incorrect for mass accuracy (all analytes contain value -10000). The error message in the prompt window "Analyte: xx is being troublesome, kill it". Error in place since the update from Python2 to Python3. Suspect that a function that required an integer parameter was called with a floating point number needs to be updated. Finally, the self.version build number should be updated (it is not updated and is now a duplicate).

**To Reproduce**
Python product version 3.8.2
1. Open 210831_FinalLaCyTools_Summing self.version = "1.1.0-alpha" build = "190207b" 
2. Select batch process, Reference File, Batch Processing, Output Format [select all]
3. Press Run Batch Process
4. Prompt window produces "Analyte: xx is being troublesome, kill it"
5. Output summary file contains the wrong ppm values

**Expected behavior**
Expect ppm values of minus and plus to be calculated.  

**Screenshots**
![image](https://user-images.githubusercontent.com/99261669/152995743-b54f3c54-87e9-46bf-99e7-8db0e97727d2.png)
![image](https://user-images.githubusercontent.com/99261669/152995646-128d6079-927e-4343-9b7d-10841deb04df.png)

**Desktop (please complete the following information):**
![image](https://user-images.githubusercontent.com/99261669/152996027-66749a95-1b5a-4b8a-b5cf-f63b01f8d792.png)

**Smartphone (please complete the following information):**

**Additional context**
When Python 3 (product version 3.7.3) is used, a warning is produced; "<class 'numpy.float64'> cannot be safely interpreted as an integer." 
Python product version 3.7.3
1. Open 210831_FinalLaCyTools_Summing self.version = "1.1.0-alpha" build = "190207b" 
2. Select batch process, Reference File, Batch Processing, Output Format [select all]
3. Press Run Batch Process
4. Prompt window produces "<class 'numpy.float64'> cannot be safely interpreted as an integer."
5.The warning message is ignored
![image](https://user-images.githubusercontent.com/99261669/152995781-1ecf1bac-da80-4d2b-b11e-3da90006c4df.png)
![image](https://user-images.githubusercontent.com/99261669/152995824-45c94a47-1dbe-4ba5-8079-89928e22e9de.png)

**Comment by Tarskin** [comment]
Cheers on including the warning, is actually quite a useful one. Could you also provide a single measurement where this occurs?

**Comment by ahi-ed** [comment]
Which file would you like me to upload?
![image](https://user-images.githubusercontent.com/99261669/152998481-c8028fe6-032a-48ac-a33d-ebd63f9db139.png)

**Comment by Tarskin** [comment]
Yeah, it won't allow it on here because of the size, but you can send me a single mzXML that shows the problem (through a specialized service) to my regular mail.

**Comment by Tarskin** [comment]
Issue has been solved in latest commit on the branch, leaving issue open until the 11th of February to ensure that there are no remaining issues.
